libs/shared/src/entities/tenant/tenant.interface.ts

```diff
@@ -0,0 +1,21 @@
+import { EnvironmentId, TenantCustomData } from '../../types';
+
+export interface ITenantEntity {
+  _id?: TenantId;
+
+  identifier: string;
+
+  name?: string;
+
+  deleted?: boolean;
+
+  createdAt: string;
+
+  updatedAt: string;
+
+  data?: TenantCustomData;
+
+  _environmentId: EnvironmentId;
+}
+
+export type TenantId = string;
```

Comment:
You could add that TenantId type to the types folder as well as you already have one for `TenantCustomData`

Reply:
Good point, done :)